pkg/command/context.go

```diff
@@ -492,10 +492,17 @@ func globalUCPLogin(c *configtypes.Context) error {
 	if err != nil {
 		return err
 	}
+	// update the current context in the kubeconfig file after creating the context
```

Comment:
@vuil Added logic to update the current kubecontext in the kubeconfig to the kube context created as result of creating CLI context for Kubernetes-cluster-endpoint(TKG with pinniped)

Reply:
@prkalle nit(preexisting): probably better to put the skipsync check in syncCurrentKubeContext

Reply:
Agreed. Moved skipsync check to syncCurrentKubeContext() method.